src/main/scala/cromwell/engine/workflow/WorkflowManagerActor.scala

```diff
@@ -314,11 +314,13 @@ class WorkflowManagerActor(backend: Backend) extends LoggingFSM[WorkflowManagerS
       BackendCallKey(descriptor.namespace.workflow.findCallByName(callName).get, key.index, key.attempt)
     def backendCallFromKey(descriptor: WorkflowDescriptor, callName: String, key: ExecutionDatabaseKey) =
       backend.bindCall(descriptor, callKey(descriptor, callName, key))
+
+    import cromwell.engine.finalcall.FinalCall._
```

Comment:
It was more a scoping thing - since this imports an implicit on String, having it widely available seemed nasty to me